collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
I'm trying something now. I will store the collectorId in options when the team selection is "Any". It will then be passed into the feature rest calls which will have two service variants: get*ByComponentId and get*ByCollectorId. The get*ByComponentId will simply extract the collectorId from the component and call the corresponding get*ByComponentId